hack/quick-install.sh

```diff
@@ -27,11 +29,11 @@ EOF
 }
 
 delete() {
+  helm delete karpenter || true
```

Comment:
nice catch! I'm a bit shocked that kube-prometheus-stack doesn't delete child resources via k8s garbage collection.